spec/client_spec.rb

```diff
@@ -30,7 +30,7 @@
 
     before(:each) do
       stub_request(:head, 'http://127.0.0.1:4000/acme/new-reg').with(
-        headers: { 'Accept' => '*/*', 'User-Agent' => 'Faraday v0.9.2' }
+        headers: { 'Accept' => '*/*', 'User-Agent' => 'Acme::Client v0.3.6 (https://github.com/unixcharles/acme-client)' }
```

Comment:
Ah, good point. I had fixed that in an initial draft and then hard coded to simplify debugging. I forgot to set it back. Fixed.